Makefile

```diff
@@ -66,6 +66,15 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: vendor
+vendor:
+	go mod tidy
+	go mod vendor
+
+.PHONY: check
+check: vendor fmt vet
```

Comment:
this is IMHO confusing. 

I would never assume `check` means check vendoring has no changes. 

Can you name this `make check-vendoring` instead ? I can follow it up with a PR if you prefer.

Reply:
sure will create new PR

Reply:
https://github.com/kubevirt/ipam-extensions/pull/98